snappy/.gitignore

```diff
@@ -0,0 +1,12 @@
+# Version Tracking Ignore Rules for Snappy
+# https://git-scm.com/docs/gitignore
+## Ignore built snap packages
+*.snap
+
+## Ignore stages autogeneraged files
+/*/parts/
```

Comment:
`.gitignore` in a subdirectory should use relative, not absolute (starting with `/`) paths to ignorable files...

Reply:
Refer to gitignore(5):

Patterns read from a .gitignore file in the same directory as the path, or in any parent directory, with patterns in the higher level files (up to the toplevel of the work tree) being overridden by those in lower level files down to the directory containing the file. **These patterns match relative to the location of the .gitignore file.** A project normally includes such .gitignore files in its repository, containing patterns for files generated as part of the project build.

I also have confirmed that the ignore rules are effective.